cypress/e2e/gui/gui-engage-sphere.spec.cy.js

```diff
@@ -0,0 +1,190 @@
+
+            cy.get('tbody tr').should('have.length', 1);
+        });
+
+        it('filters the customers by retail industry', () => {
+            cy.intercept('GET', `${CUSTOMERS_API_URL}?page=1&limit=10&size=All&industry=Retail`, 
+              { fixture: 'industry/retail' }
+            ).as('getRetailCustomers');
+            cy.get('[data-testid="industry-filter"]').select('Retail');
+            cy.wait('@getRetailCustomers');
+
+            cy.get('tbody tr').should('have.length', 1);
+        });
+
+        it('filters the customers by technology industry', () => {
+            cy.intercept('GET', `${CUSTOMERS_API_URL}?page=1&limit=10&size=All&industry=Technology`,
+              { fixture: 'industry/technology'}
+            ).as('getTechnologyCustomers');
+            cy.get('[data-testid="industry-filter"]').select('Technology');
+            cy.wait('@getTechnologyCustomers');
+
+            cy.get('tbody tr').should('have.length', 3);
+        });
+
+        it('filters the customers by HR industry', () => {
+            cy.intercept('GET', `${CUSTOMERS_API_URL}?page=1&limit=10&size=All&industry=HR`,
+              { fixture: 'industry/hr' }
+            ).as('getHRCustomers');
+            cy.get('[data-testid="industry-filter"]').select('HR');
+            cy.wait('@getHRCustomers');
+            cy.get('tbody tr').should('have.length', 4);
+        });
+
+        it('filters the customers by finance industry', () => {
+            cy.intercept('GET', `${CUSTOMERS_API_URL}?page=1&limit=10&size=All&industry=Finance`,
+              { fixture: 'industry/finance'}
+            ).as('getFinanceCustomers');
+            cy.get('[data-testid="industry-filter"]').select('Finance');
+            cy.wait('@getFinanceCustomers');
+            cy.get('tbody tr').should('have.length', 6);
+        });
+
+        it('filters the customers by all industries', () => {
+            cy.get('[data-testid="industry-filter"]').select('Technology');
+            cy.intercept('GET', `${CUSTOMERS_API_URL}?page=1&limit=10&size=All&industry=All`,
+              { fixture: 'all' }
+            ).as('getAllCustomers');
+            cy.get('[data-testid="industry-filter"]').select('All');
+            cy.wait('@getAllCustomers');
+            cy.get('tbody tr').should('have.length', 9);
+        });
+    });
+
+    context('View', () => {
+        it('goes back to the customers list when clicking the "Back" button', () => {
+            cy.contains('button', 'View').eq(0).click();
+            cy.contains('button', 'Back').click();
+            cy.contains('p', 'Below is our customer list.').should('be.visible');
+        });
+
+        it('shows a Loading... fallback element before the initial customers fetch', () => {
+            cy.reload();
+            cy.contains('p', 'Loading...')
+                .should('be.visible');
+        });
+    });
+
+
+    context('Acessibility', () => {
+        beforeEach(() => {
+            cy.injectAxe();
```

Comment:
```suggestion
            cy.injectAxe();
            cy.get('[data-theme="light"]').should('exist');
```

Que tal garantir que está mesmo no modo claro antes de começar?